libcontainer/rootfs_linux.go

```diff
@@ -240,7 +240,7 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		// any previous mounts can invalidate the next mount's destination.
 		// this can happen when a user specifies mounts within other mounts to cause breakouts or other
 		// evil stuff to try to escape the container's rootfs.
-		if dest, err = symlink.FollowSymlinkInScope(dest, rootfs); err != nil {
```

Comment:
The correct ordering of arguments would be:

```
if dest, err = securejoin.SecureJoin(rootfs, m.Destination); err != nil {
```

However it feels like this section would be a better place to fix:

```
164 var (
165  	dest = m.Destination
166 )
167 if !strings.HasPrefix(dest, rootfs) {
168 	dest = filepath.Join(rootfs, dest)
169 }
```

And then the two calls for `symlink.FollowSymlinkInScope` shouldn't be necessary.

/cc @crosbymichael

Reply:
ohhh :sweat_smile: